DM-36616: Make a Getting Started Guide For Analysis Tools (#79)
lsst/analysis_tools: doc/lsst.analysis.tools/plot-types.rst

> Plot Types
> ==========
> Details of current plot types are here, these plots are all in ``python/lsst/analysis/tools/actions/plot`` and

Reviewer: ```suggestion
##########
Plot Types
##########

This page lists the plot types provided by ``analysis_tools``.
These plots's modules are located in the ``python/lsst/analysis/tools/actions/plot`` directory of the repository and
```

> can all be found on `github <https://github.com/lsst/analysis_tools/blob/main/python/lsst/analysis/tools/actions/plot/>`__.

Reviewer: ```suggestion
can all be found on `GitHub <https://github.com/lsst/analysis_tools/blob/main/python/lsst/analysis/tools/actions/plot/>`__.
```

> .. automodapi:: lsst.analysis.tools.actions.plot
>    :no-inherited-members:

Reviewer: ```suggestion
.. automodapi:: lsst.analysis.tools.actions.plot
   :no-inherited-members:
   :noindex:
```

This this; I'm not certain it'll work but might solve the warnings about duplicate documentation.

Author: It doesn't solve the warnings and it breaks things elsewhere. I'm not sure what to do about this, but if it won't break anything I will merge it and fix it later.
